src/components/label-pill/LabelPill.stories.md

```diff
@@ -0,0 +1,3 @@
+`import LabelPill from 'box-ui-elements/es/components/label-pill/LabelPill';`
+
+_Note_: this component is meant to be used with one of the formatted strings produced by react-intl (`FormattedMessage`, `FormattedRelativeDate`, etc.) **or** one of our icon assets.
```

Comment:
Still need the note?

Reply:
this is still true, tho there's nothing to enforce it. If someone tries to put different content in this component, it may not have a good layout.